lib/Jsrt/JsrtHelper.cpp

```diff
@@ -89,7 +89,7 @@ void JsrtCallbackState::ObjectBeforeCallectCallbackWrapper(JsObjectBeforeCollect
         pthread_key_create(&s_threadLocalDummy, DISPOSE_CHAKRA_CORE_THREAD);
 #endif
 
-#if defined(CHAKRA_STATIC_LIBRARY)
+#if defined(CHAKRA_STATIC_LIBRARY) || !defined(_WIN32)
```

Comment:
> Basically it was failing (other host with libChakraCore.so) until I enabled this part. 

We shouldn't try to fix shared library issues by enabling static lib initialization.

> In the case of ch, we have manual PAL_Init... call in wmain. Other hosts only call JSRT.

Actually, my plan was to remove PAL_Init.. call entirely. Similar to SysInfo update, we will initialize PAL part lazily.  My guess is that we will have WinBash is also working after that. However, for now, the only option we have is to use static lib on other host until we cleanup that part or initialize PAL as we do with ch. Enabling static lib features on non-Windows no matter what doesn't sound like a good idea.